ra/proto/ra.proto

```diff
@@ -70,6 +70,8 @@ message AdministrativelyRevokeCertificateRequest {
 message NewOrderRequest {
   int64 registrationID = 1;
   repeated string names = 2;
+  string replaces = 3;
+  bool limitsExempt = 4;
```

Comment:
Perhaps add a comment here too that this exists because of ARI.

Reply:
I can't find anywhere where we've done similar so I'm inclined to pass on this particular request.